config/runtime.exs

```diff
@@ -82,6 +82,8 @@ base_oban_conf = [repo: DB.Repo]
 oban_crontab_all_envs = [
   {"0 */6 * * *", Transport.Jobs.ResourceHistoryDispatcherJob},
   {"30 */6 * * *", Transport.Jobs.GtfsToGeojsonConverterJob},
+  # every 6 hours but not at the same time as other jobs
```

Comment:
Comme c'est gourmand, je me décale par rapport aux autres taches qui tournent toutes les 6h, histoire de ne pas tout faire en même temps...

Reply:
Excellent ; dans l'esprit c'est proche de la notion d'un "jitter" (https://aws.amazon.com/builders-library/timeouts-retries-and-backoff-with-jitter/).

Reply:
intéressant, merci pour le lien :+1: